docker-for-mac/release-notes.md

```diff
@@ -13,13 +13,43 @@ This page contains information about the new features, improvements, known issue
+> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will be available automatically as delta updates from the previous version. When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. Whenever convenient, you can start the download of the update in the background. When the download finishes, all you need to do is to click **Update and restart** to install the latest update.
 {: .important }
 
+## Docker Desktop 3.3.0
+2021-04-08
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Upgrades
+
+- [Docker Compose 1.29.0](https://github.com/docker/compose/releases/tag/1.29.0)
+- [Compose CLI v1.0.12](https://github.com/docker/compose-cli/tree/v1.0.12)
+- [Linux kernel 5.10.25](https://hub.docker.com/layers/docker/for-desktop-kernel/5.10.25-6594e668feec68f102a58011bb42bd5dc07a7a9b/images/sha256-80e22cd9c9e6a188a785d0e23b4cefae76595abe1e4a535449627c2794b10871?context=repo)
+- [Snyk v1.461.0](https://github.com/snyk/snyk/releases/tag/v1.461.0)
+- [Docker Hub Tool v0.3.1](https://github.com/docker/hub-tool/releases/tag/v0.3.1)
+- [containerd v1.4.4](https://github.com/containerd/containerd/releases/tag/v1.4.4)
+- [runc v1.0.0-rc93](https://github.com/opencontainers/runc/releases/tag/v1.0.0-rc93)
+
+### Bug fixes and minor changes
+
+- Fixed an issue related to docker-compose action. Fixes [docker/for-win#10564](https://github.com/docker/for-win/issues/10564).
+- Fixed launching Docker Desktop from the Dock. Fixes [docker/for-mac#5515](https://github.com/docker/for-mac/issues/5515).
+- Ensure `--add-host host.docker.internal:host-gateway` causes `host.docker.internal` resolves to the host IP, rather than the IP of the IP router. See [docker/for-linux#264](https://github.com/docker/for-linux/issues/264).
+- Prevent inter-container HTTP traffic being forwarded via the HTTP proxy to the host. Fixes [docker/for-mac#5476](https://github.com/docker/for-mac/issues/5476).
+- Prevent the deletion of files which have been added to the folder containing the VM disk image. Fixes [docker/for-mac#5486](https://github.com/docker/for-mac/issues/5486).
+- Fixed an issue where delta downloads caused an `Illegal instruction exception`. Fixes [docker/for-mac#5459](https://github.com/docker/for-mac/issues/5459).
+- Apply domain-based HTTPS proxy `no_proxy` rules for encrypted connections by using TLS SNI. Fixes [docker/for-mac#2732](https://github.com/docker/for-mac/issues/2732).
+- Fixed an issue where the dashboard opened the container in a browser with port 0.
+- Perform a filesystem check when starting the Linux VM.
+- Fixed an issue when pulling an image from Docker Hub using the Docker Desktop dashboard.
```

Comment:
What issue? The user can't know from this whether their bug is fixed.

Reply:
I believe it was just not working, I have no idea if there was an error message and I don’t think there was any user report.
@trungutt ?

Reply:
@gtardif reported this behavior, we didn’t have any error message and it didn’t work (nothing happen when clicking on Pull). We have a Bugsnag report, but no user complains about thus bug

Reply:
OK, but can we at least describe briefly what the symptom was. "We fixed in issue" isn't helpful.

Reply:
What about: `Fixed an issue where pulling an image from Docker Hub using the Docker Desktop dashboard was failing silently.`?

Reply:
I went with ☝️, @stephen-turner is that OK?

Reply:
Yes, that's fine, thanks.